packages/client/hmi-client/src/components/tabs/TabContainer.vue

```diff
@@ -86,7 +89,11 @@ watch(newActiveTab, (index) => {
 			<template #tabIcon>
 				<component :is="icon"></component>
 			</template>
-			<component class="tab-content" :is="componentToRender" v-bind="tab.props"></component>
+			<component
+				class="tab-content"
+				:is="tab.props?.resourceRoute ? ResourcesListConfig : componentToRender"
```

Comment:
No specific suggestion here, but I suspect this code will be harder to maintain if it can conditionally render several different components in this TabContainer, without the benefits of static analysis or actually importing the components.

We should consider refactoring to potentially use Vue `slot`s or `teleport`s.

Reply:
Not purpose of this PR